assets/src/graph/cdClusters.graphql

```diff
@@ -30,6 +30,14 @@ fragment NodeMetricFragment on NodeMetric {
   window
 }
 
+fragment NodePoolFragment on NodePool {
```

Comment:
You don't actually need to put "Fragment" at the end of of fragment names. GQL Codegen makes this into a type named `NodePoolFragmentFragment`

Reply:
Ok thanks. I will probably need to refactor this graphql code later to reorganize where fragments are located.